### PR TITLE
Add support for specifying signing secrets on a per-request basis

### PR DESCRIPTION
**Pull request by gabrielmdeal**
For services that handle requests from multiple Slack applications.

Example:
```ruby
Slack::Events::Request.new(http_request,
                           signing_secret: signing_secret,
                           signature_expires_in: signature_expires_in))
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,6 +506,13 @@ slack_request = Slack::Events::Request.new(http_request)
 slack_request.verify!
 ```
 
+To specify secrets on a per-request basis:
+```ruby
+Slack::Events::Request.new(http_request,
+                           signing_secret: signing_secret,
+                           signature_expires_in: signature_expires_in))
```
Comment (dblock) [inline README.md]: Extra `)`

File: lib/slack/events/request.rb [hunk 2]
```diff
@@ -5,10 +5,16 @@ class MissingSigningSecret < StandardError; end
       class TimestampExpired < StandardError; end
       class InvalidSignature < StandardError; end
 
-      attr_reader :http_request
+      attr_reader :http_request,
+                  :signing_secret,
+                  :signature_expires_in
 
-      def initialize(http_request)
+      def initialize(http_request,
```
Comment (dblock) [inline lib/slack/events/request.rb]: Can we just use `options = {}` for consistency? I forget in which Ruby version the named options start getting supported, but I'd like it to be the same across the entire library (another PR can change it everywhere else).

**Comment by gabrielmdeal** [comment]
Thanks for your attention!  I have made both of those improvements.

File: README.md [hunk 3]
```diff
@@ -506,6 +506,13 @@ slack_request = Slack::Events::Request.new(http_request)
 slack_request.verify!
 ```
 
+To specify secrets on a per-request basis:
+```ruby
+Slack::Events::Request.new(http_request,
+                           signing_secret: signing_secret,
+						   signature_expires_in: signature_expires_in)
+```
```
Comment (dblock) [inline README.md]: Something's odd with tabs/spaces here, please fix?

**Comment by dblock** [comment]
Merged

**Pull request merged**